torchaudio/datasets/utils.py

```diff
@@ -189,3 +191,15 @@ def extract_archive(from_path: str, to_path: Optional[str] = None, overwrite: bo
         pass
 
     raise NotImplementedError("We currently only support tar.gz, tgz, and zip achives.")
+
+
+def load_waveform(
```

Comment:
```suggestion
def _load_waveform(
```

I am personally, not sure if this will be very reusable, but let's see.